people.json

```diff
@@ -0,0 +1 @@
+[{"type":"Student","name":"Sam","id":908,"age":24,"classroom":"FR","parent_permission":"true"},{"type":"Teacher","name":"DD","id":819,"age":43,"specialization":"HYD"}]
```

Comment:
- [ ] I've been checking your app and it seems that your `.json` files have been broken with your last changes. Kindly remove the `.json` files where you store the data, so it can be created again correctly.
- [ ] I don't have access to your `person.rb` file for review, so will use this line to review it. Kindly in your `person.rb` file fix the bug where you need 3 arguments based on the flow of your app. In your class person, you have 2 arguments + `parent_permision = true` as default, which is causing this problem. Maybe setting `parent_permision` to only an argument instead of a default value fixes it. At least for me, it works.

![image](https://github.com/aima98/OOP-School-Library1/assets/93630700/51aa0fed-7811-4585-a49c-7381809ab6b9)